src_erl/Communication_Layer/http_Nerlserver/src/newparser.erl

```diff
@@ -0,0 +1,121 @@
+%%%-------------------------------------------------------------------
+%%% @author kapelnik
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 19. Apr 2021 4:27 AM
+%%%-------------------------------------------------------------------
+-module(newparser).
+-author("kapelnik").
+
+%% API
+-export([parse_file/0]).
+
+
+%%parsing a given CSV file
+parse_file() ->
+  io:format("~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n~n",[]),
```

Comment:
remove redundant printing - such a code can't merged into master. 
You can wrap it with debug flag section or something